script.js

```diff
@@ -0,0 +1,34 @@
+$(function(){
+    var prefix = "https://cors-anywhere.herokuapp.com/";
+    var tweetLink = "https://twitter.com/intent/tweet?text=";
+    var quoteUrl = "https://quotesondesign.com/wp-json/posts?filter[orderby]=rand&filter[posts_per_page]=1";
+    
+    function getQuote() {
+        $.getJSON(prefix + quoteUrl, createTweet);
+        $.ajaxSetup({cache: false});
+}
+    function createTweet(input) {
+        var data = input[0];
```

Comment:
before you sign this value to new variable check if input is an array with at least one item